app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,879 @@
+    twoWay: true,                   // Is two-way               
+    key: 'reviews',                 // Attribute key                    
+    twoWayKey: 'movie',             // Two-way attribute key            
+    onDelete: 'cascade',            // On delete action                 
+);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Kotlin</h3>
+        <div class="ide margin-top-small" data-lang="kotlin" data-lang-label="Kotlin SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Databases
+
+val client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+    .setProject("5df5acd0d48c2")                // Your project ID
+
+val databases = Databases(client)
+
+databases.createRelationshipAttribute(
+    databaseId = "marvel",           // Database ID                 
+    collectionId = "movies",         // Collection ID                      
+    relatedCollectionId = "reviews", // Related collection ID               
+    type = "oneToMany",              // Relationship type           
+    twoWay = true,                   // Is two-way              
+    key = "reviews",                 // Attribute key           
+    twoWayKey = "movie",             // Two-way attribute key       
+    onDelete = "cascade"             // On delete action        
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Swift</h3>
+        <div class="ide margin-top-small" data-lang="swift" data-lang-label="Swift SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+    .setProject("5df5acd0d48c2")                // Your project ID
+
+let databases = Databases(client)
+
+databases.createRelationshipAttribute(
+    databaseId: "marvel",           // Database ID              
+    collectionId: "movies",         // Collection ID            
+    relatedCollectionId: "reviews", // Related collection ID    
+    type: "oneToMany",              // Relationship type        
+    twoWay: true,                   // Is two-way               
+    key: "reviews",                 // Attribute key            
+    twoWayKey: "movie",             // Two-way attribute key        
+    onDelete: "cascade"             // On delete action         
+)</code></pre>
+        </div>
+    </li>
+</ul>
+
+<p>The above example adds a relationship between the collections <b>movies</b> and <b>reviews</b>. A relationship attribute with the key <code>reviews</code> is added to the movies collection and another relationship attribute with the key <code>movie</code> is added to the reviews collection.</p>
+
+<p>Relationship attribute keys are used to reference the related collection in queries, so pick something that's intuitive and easy to remember.</p>
```

Comment:
This sentence is being repeated here

https://github.com/appwrite/docs/pull/342/files#diff-6eed692b8191fbcdb9de18d7a355dd6d02a79ba9a8b18f092fe7b2b20c373f38R105

![Screenshot 2023-04-07 at 4 53 49 PM](https://user-images.githubusercontent.com/20852629/230612156-d6164cee-feb9-4884-b18e-215c761e2eee.png)

We could remove this occurrence or rewrite it

Reply:
Done